docs/source/tutorial/firststeps.rst

```diff
@@ -173,7 +175,7 @@ The sample test bundle above requires a network inventory and a running network
 
 How it works: Each test module implements a context class to provide module-specific functionality to its tests. This context class is a  ``NutsContext`` or a subclass of it. This guarantees a consistent interface across all tests for test setup and execution. 
 
-The predefined test classes which depend on a network all use `nornir <https://nornir.readthedocs.io/en/latest/>`__ in order to communicate with the network devices. Those test classes all derive all from a more specific ``NornirNutsContext``, which provides a nornir instance and nornir-specific helpers. 
+The predefined test classes which depend on a network all use `nornir <https://nornir.readthedocs.io/en/latest/>`__ in order to communicate with the network devices. Those test classes all derive from a more specific ``NornirNutsContext``, which provides a nornir instance and nornir-specific helpers. 
```

Comment:
is that still true with the merge of the SA in 3.1?

Reply:
Good point. I need to check the tests from the SA